config/config.go

```diff
@@ -69,15 +71,16 @@ type Config struct {
 	BaseConfig `mapstructure:",squash"`
 
 	// Options for services
-	RPC             *RPCConfig             `mapstructure:"rpc"`
-	P2P             *P2PConfig             `mapstructure:"p2p"`
-	Mempool         *MempoolConfig         `mapstructure:"mempool"`
-	StateSync       *StateSyncConfig       `mapstructure:"statesync"`
-	FastSync        *FastSyncConfig        `mapstructure:"fastsync"`
-	Consensus       *ConsensusConfig       `mapstructure:"consensus"`
-	Storage         *StorageConfig         `mapstructure:"storage"`
-	TxIndex         *TxIndexConfig         `mapstructure:"tx_index"`
-	Instrumentation *InstrumentationConfig `mapstructure:"instrumentation"`
+	RPC             *RPCConfig                   `mapstructure:"rpc"`
+	P2P             *P2PConfig                   `mapstructure:"p2p"`
+	Mempool         *MempoolConfig               `mapstructure:"mempool"`
+	StateSync       *StateSyncConfig             `mapstructure:"statesync"`
+	FastSync        *FastSyncConfig              `mapstructure:"fastsync"`
+	Consensus       *ConsensusConfig             `mapstructure:"consensus"`
+	Storage         *StorageConfig               `mapstructure:"storage"`
+	TxIndex         *TxIndexConfig               `mapstructure:"tx_index"`
+	Instrumentation *InstrumentationConfig       `mapstructure:"instrumentation"`
+	EventCollector  *remote.EventCollectorConfig `mapstructure:"event_collector"`
```

Comment:
kinda just threw these names together, so I'm definitely open to better names. I'm leaning towards just calling it an influxdb client, because that's basically what it is

Reply:
The naming looks good to me, but it may be beneficial to include a brief comment about the service it provides.

Reply:
It could fall under instrumentation, but I'm viewing trace level data collection as much more arbitrary and specific atm. Its not really something we use to get a glance at what is happening at the network, its something that we use in a more one-off way to debug really specific issues. We can include in it in that config though

Reply:
>  Its not really something we use to get a glance at what is happening at the network

Why not? :) 

Aren't you using it now to work out how often we have multi-round heights in our testnets

Anyway, this strikes me as `Instrumentation`

Reply:
> Anyway, this strikes me as Instrumentation 

okay cool, will add to the instumentation config instead :+1: 

> Why not? :)

I think this comes down to how much detail is being extracted or summarized. for example, if we were simply wanting to view how many blocks took multiple rounds to come to consensus, then we could do that in a simple light way using the existing metrics to store a summary of the data. That summary is useful to "glance" at. However, if we wanted to examine why those heights are taking multiple rounds, then we would need something not significantly less summarized and takes more time to analyze.

Reply:
also, when we store the data in a detailed (as opposed to summarized) way, in order to glance at it we have to write significantly more complex queries, which we don't currently have atm and can be time consuming to write